tests/test_vibradorm.py

```
@@ -489,18 +548,36 @@ async def test_movement_sends_stop_after(
 
 
 class TestVibradormPresets:
-    """Test Vibradorm preset commands."""
+    """Test Vibradorm preset commands.
+
+    Presets use hold-to-run with 600 repeats (60s). Tests set the cancel event
+    after the first write so the loop exits quickly.
+    """
+
+    @staticmethod
+    def _cancel_after_first_write(coordinator):
```

Ruff (0.15.1)

[warning] Missing return type annotation for staticmethod `_cancel_after_first_write`

(ANN205)